site_meta.go

```diff
@@ -35,18 +58,40 @@ func parseMetaAttr(s *goquery.Selection) *MetaAttr {
 	return &attr
 }
 
-func isHTMLContent(url string) (bool, error) {
+func extractCharset(contentType string) string {
+	logger.Printf("contentType: %s\n", contentType)
+
+	exp := regexp.MustCompile(`charset=((\w|\d|\-)*)`)
+	group := exp.FindStringSubmatch(contentType)
+
+	if len(group) < 2 {
+		logger.Printf("len(group) less than 2")
+		return DefaultEncoding
+	}
+
+	// TODO: Normalize target encoding.
```

Comment:
Currently, just use matched charset which in `Content-Type` response header.

This will cause an unexpected error when matched charset is not supported by `iconv-go`.

This should be fixed in near future. 😅